src/sql/workbench/services/connection/browser/connectionDialogService.ts

```diff
@@ -153,12 +153,15 @@ export class ConnectionDialogService implements IConnectionDialogService {
 	}
 
 	private handleOnConnect(params: INewConnectionParams, profile?: IConnectionProfile): void {
+		this._logService.info('ConnectionDialogService: onConnect event is received');
```

Comment:
We should use debug for this and then pass in `--log debug` when starting up the tests so we aren't cluttering up the console outside of tests

Reply:
Since we are using 'trace' here, that should also include all debug logs right? https://github.com/microsoft/azuredatastudio/blob/5922047f6bbd2c61705f46986bcf0c19da6cb4fa/test/smoke/src/main.ts#L253

Reply:
That looks correct! Make sure to check the logs after changing the log level to make sure though